scss/mixins.scss

```diff
@@ -117,7 +120,7 @@ $_o-brand-define-readme: "https://github.com/Financial-Times/o-brand/#obranddefi
         'Cannot check if component "#{$component}" supports variant "#{$variant}"'
     );
     // Check support for a pre-defined variant.
-    $brand: _oBrandGetCurrentBrand();
+    $brand: oBrandGetCurrentBrand();
     $brand-config: _oBrandGetConfig($component, $o-brand);
```

Comment:
It was already like this, but looks like we want to pass `$brand` rather than `$o-brand`.